run-tests.sh

```diff
@@ -13,22 +13,69 @@ set -o errexit
 set -o nounset
 
 
-cli_docs_url=https://raw.githubusercontent.com/reanahub/docs.reana.io/master/docs/reference/reana-client-cli-api/index.md
-docs_differ_error_msg='Current reana-client differs with the documentation. Please update http://docs.reana.io/reference/reana-client-cli-api/.'
-python_version=$(python -c 'import sys;  print(sys.version_info.major)')
-
-pydocstyle reana_client
-reana-client --help > cmd_list.txt
-diff -q -w docs/cmd_list.txt cmd_list.txt
-rm cmd_list.txt
-if [ "$python_version" -eq 3 ]
-then
+check_script () {
+    shellcheck run-tests.sh
+}
+
+check_black () {
     black --check .
+}
+
+check_pydocstyle () {
+    pydocstyle reana_client
+}
+
+check_manifest () {
+    check-manifest
+}
+
+check_cli_cmds () {
+    reana-client --help > cmd_list.txt
+    diff -q -w docs/cmd_list.txt cmd_list.txt
+    rm cmd_list.txt
+}
+
+check_cli_api () {
+    cli_docs_url=https://raw.githubusercontent.com/reanahub/docs.reana.io/master/docs/reference/reana-client-cli-api/index.md
+    docs_differ_error_msg='Current reana-client differs with the documentation. Please update http://docs.reana.io/reference/reana-client-cli-api/.'
     python scripts/generate_cli_api.py > cli_api.md
-    #(diff -q -w  cli_api.md <(curl -s $cli_docs_url) || (echo $docs_differ_error_msg && exit 1))
+    (diff -q -w  cli_api.md <(curl -s $cli_docs_url) || (echo "$docs_differ_error_msg" && exit 1))
     rm cli_api.md
+}
+
+check_sphinx () {
+    sphinx-build -qnNW docs docs/_build/html
+    sphinx-build -qnNW -b doctest docs docs/_build/doctest
+}
+
+check_pytest () {
+    python setup.py test
+}
+
+
+if [ $# -eq 0 ]; then
+    check_script
```

Comment:
Just a thought: maybe it would be cleaner to have another function and call it in case of no arguments?
```bash
check_all() {
    check_script
    check_black
    check_pydocstyle
    check_manifest
    check_cli_cmds
    check_cli_api
    check_sphinx
    check_pytest
}
```
what do you think?

Reply:
Yes, I can do that 👍

Reply:
Done 👍